isthisstockgood/Active/Zacks.py

```diff
@@ -0,0 +1,42 @@
+import re
+
+class Zacks:
+    def __init__(self, ticker_symbol):
+        base_url = "https://www.zacks.com/stock/quote"
+
+        self.url = f"{base_url}/{ticker_symbol}/detailed-earning-estimates"
+        self.ticker_symbol = ticker_symbol
+        self.five_year_growth_rate = None
+        self.maintenance_capital_expenditures = None
+
+    def parse(self, response, **kwargs):
+        if response.status_code != 200:
+          print(f"{response.status_code}: {response.text}")
```

Comment:
This is the point where in the future we could create a GitHub issue right away, adding `response.text` to the description.

For now we can either leave the `print` statements here, or remove them if they do not add any value.

Reply:
Removed for now.